src/schemas/index.js

```diff
@@ -32,11 +32,15 @@ export const schema = {
     },
     'host': {
       'type': 'link',
-      'target': '/header/info/host'
+      'target': '/header/host/host'
     },
     'basePath': {
       'type': 'link',
-      'target': '/header/info/basePath'
+      'target': '/header/host/basePath'
```

Comment:
What does the 'target' field do? I think we should treat URLs as case insensitive, so use hyphens instead of camel case.